Add Aspose.PDF for .NET 24.2 release notes (#176)
Aspose/releases.aspose.com: content/en/pdf/net/release-notes/2024/aspose-pdf-for-net-24-02-release-notes.md

---
id: "aspose-pdf-for-net-24-2-release-notes"
slug: "aspose-pdf-for-net-24-2-release-notes"
linktitle: "Aspose.PDF for .NET 24.2"
title: "Aspose.PDF for .NET 24.2"
weight: 120
description: "This page contains new Aspose.PDF for .NET features, enhancement, and bug fixes in 2024, version 24.2."
type: "repository"
layout: "release"
family_listing_page_title: "Aspose.PDF for .NET 24.2"
lastmod: "2024-02-15"
sitemap:
  changefreq: "monthly"
  priority: 0.7
---

{{% alert color="primary" %}}

This page contains release notes information for Aspose.PDF for .NET 24.1.

{{% /alert %}}

## Improvements and Changes

|**Key**|**Summary**|**Category**|
| :- | :- | :- |

> |PDFNET-48540|Include the current page object in the CallBackGetHocr delegate|Feature|

Reviewer: ```suggestion
|PDFNET-48540|Add reference of the current page to the CallBackGetHocr delegate|Feature|
```

|PDFNET-54667|Getting the vector data from a PDF file|Feature|
|PDFNET-34126|Converting Searchable PDF to XPS (Keeping Text Selectable in resultant XPS)|Feature|
|PDFNET-56200|Optimize image usage by PDF to Markdown converter|Feature|
|PDFNET-56439|Passing Page as an image to HOCR Callback method|Feature|
|PDFNET-53356|Add the ability to load DjVu format files|Feature|
|PDFNET-54734|Add the ability to load CDR format files|Feature|
|PDFNET-55965|Extending the MdLoadOptions to support CSS and PageInfo|Enhancement|

> |PDFNET-56430|Cannot find a substitution for PrinterSettings.PaperSources|Enhancement|

Reviewer: ```suggestion
|PDFNET-56430|Perform a correct substitution for PrinterSettings.PaperSources|Enhancement|
```

|PDFNET-42905|Support setting review status for comments in TextAnnotation|Enhancement|
|PDFNET-56204|Extract higher quality images in PDF to Markdown converter|Enhancement|
|PDFNET-55738|Providing users the opportunity to determine the results of constructing (compiling) LaTeX documents|Enhancement|

> |PDFNET-56373|On converting HTML to PDF margins are not set from CSS|Enhancement|

Reviewer: ```suggestion
|PDFNET-56373|Enable control of page settings source during conversion of HTML(CSS) to PDF|Enhancement|
```
